metadata.json

```diff
@@ -10,15 +10,15 @@
   "dependencies": [
     {
       "name": "puppetlabs-stdlib",
-      "version_requirement": ">= 1.0.0"
+      "version_requirement": ">= 1.0.0 < 7.0.0"
     },
     {
       "name": "camptocamp/systemd",
-      "version_requirement": ">= 1.1.1"
+      "version_requirement": ">= 1.1.1 < 1.2.0"
     },
     {
       "name": "puppet-archive",
-      "version_requirement": ">= 1.1.2"
+      "version_requirement": ">= 1.1.2 < 1.2.0"
     }
   ],
```

Comment:
I don't know what would be the better upper bounds here
I can go as it is done with the minimum changes, just accepting the patches updates. It can be done also allowing more versions 

```json
"dependencies": [
  {
    "name": "puppetlabs-stdlib",
    "version_requirement": ">= 1.0.0 < 7.0.0"
  },
  {
    "name": "camptocamp/systemd",
    "version_requirement": ">= 1.1.1 < 3.0.0"
  },
  {
    "name": "puppet-archive",
    "version_requirement": ">= 1.1.2 < 5.0.0"
  }
],
```

More info:
- https://puppet.com/docs/puppet/4.10/modules_metadata.html#version-specifiers-in-module-metadata
- https://forge.puppet.com/modules/puppet/archive
- https://forge.puppet.com/modules/camptocamp/systemd
- https://forge.puppet.com/modules/puppetlabs/stdlib

Reply:
There is just one usage of archive and systemd::unit_file resources in manifests/instance.pp... Maybe checking that they have the same signature in the latest version would be enough. WDYT?

Reply:
I've been checking, and they look compatible. At least, with the resources that are used in `manifests/instance.pp`